docs/pages/repo/docs/ci.mdx

```diff
@@ -5,10 +5,24 @@ description: Recipes for using Turborepo with GitHub Actions, CircleCI, and othe
 
 # Continuous Integration
 
-Turborepo not only speeds up builds, but also your CI pipeline. Below are a few ways to use Turborepo with various Continuous Integration providers.
+Turborepo not only speeds up builds, but also the rest of your CI pipeline by using [Remote Caching](/repo/docs/core-concepts/remote-caching). Below are a few platform recipes to use Turborepo with your Continuous Integration providers.
 
-- [Vercel](/repo/docs/ci/vercel)
+- [Vercel](/repo/docs/ci/vercel) (zero-configuration required)
```

Comment:
```suggestion
- [Vercel](/repo/docs/ci/vercel) (zero configuration required)
```

The hyphen makes it sound backwards -- that "xxx" is required, where "xxx" is something called "zero configuration". You want to say the opposite, that _nothing_ is required.